documentation/feature/medical-system.md

```diff
@@ -125,83 +144,101 @@ Advanced medical system brings more different types of wounds, each with its own
 
-+#### 3.1.7 PAK and Surgical kit
-Using Personal Aid Kit brings patient to the best possible health state. Depending on module options, PAK can be used only in specified situations, specified place and by specified person.
+#### 3.1.7 PAK and Surgical kit
+
+Using Personal Aid Kit brings patient to the best possible health state. Depending on module options, PAK can be used only in specified situations, such as when all wounds are bandaged and patient is awake, specified place, such as medical vehicle or building, and by specified person, such as medic or doctor.
+
 Surgical kit is used to stich bandaged wounds so they will never open again. Depending on module options, surgical kit can be used only in specified situations, specified place and by specified person.
 
-### 3.2 Advanced medical system - recommended gear
- 
+### 3.2 Healing guideline
+
+Below you will find how to handle wounds if you play as a soldier or a medic.
+
+#### 3.2.1 Healing yourself as a soldier
+
+Follow steps below.
+
+ 1. Are you bleeding? If yes go to step 2, if no, go to step 5.
+ 2. Did your limbs got hit? If yes, apply tourniquet on them.
+ 3. Start applying proper bandages on your wounds.
+ 4. Remember to take off your tourniquet!
+ 5. Do you feel pain? If no, skip step below.
+ 6. Check your heart rate, if it's normal, apply morphine. You should be fine now.
+ 7. Always call for a medic if your wounds were severe. Tell him what wounds you sustained and what medications you used.
+
+#### 3.2.2 Healing other soldiers as a medic
+
+Follow steps below.
+
+ 1. Most important is to check if patient is conscious and how many wounds he sustained. Quickly check pulse and blood pressure. If there is no pulse, go to next step, if there is pulse but patient is unconscious, then you should do everything to stabilize patient status and he will wake up on his own after a while.
+ 2. Quickly start bandaging his wounds, always use proper bandages vs given wound type, call for a help, tell your assistant to do CPR while you are bandaging. If there are many wounds, apply tourniquet on wounded limbs. **Do not apply any medications** while patient has **no heart rate**! Wait until he wakes up from cardiac arrest! However, **if there is pain and patient is only unconcious and has heart rate**, apply morphine as too much pain will make him stay unconcious for a very long time.
+ 3. If patient blood pressure is low i.e. under 100/80 then you should apply 1000ml of IV. Tranfusion will take some time to complete and you should monitor patient health status until it finishes.
+ 4. If cariac arrest was sustained and CPR was successful, immediately check patient heart rate, if its low, apply epinephrine to raise it again.
+ 5. **If morphine was not applied before**, when heart rate raises to safe level (around 80BPM), apply morphine to counter pain.
+ 6. If patient heart rate is very high i.e. over 140, apply atropine as soon as possible.
```

Comment:
This is not necessarily true; it depends on the situation and all kinds of other factors. Ie how much medication has been given already? Is the patient bleeding? Has the patient lost a lot of blood? Is the HR still rising or is it stable? Or lowering? 

Statements such as this are dangerous, as it might cause confusion.